scss/_post.scss

```diff
@@ -0,0 +1,183 @@
+.post{
+    position: relative;
+    display: inline-flex;
+    width: 100%;
+    &__image-left{
+        position: relative;
+        width: 15%;
+        align-items: center;
+        box-shadow: 15px 0px $green;
+        margin-right: 15px;
+    }
+    &__image-left-img{
+        object-fit: cover;
+        width: 100%;
+        height: 100%;
+    }
+    &__description{
+        width: 85%;
+        background-color: $light_grey2;
+        padding: 30px;
+    }
+    &__description-title{
+        display: flex;
+        align-items: center;
+        margin-bottom: 10px;
+    }
+    &__title{
+        font-size: 26px;
+        font-weight: 700;
+    }
+    &__btn-desktop{
+        display: none;
+    }
+    &__subtitle{
+        font-size: 12px;
+        font-weight: 500;
+        color: $grey;
+        margin-bottom: 20px;
+    }
+    &__event{
+        $this: &;
+        display: flex;
+        justify-content: space-between;
+        align-items: top;
+        border-bottom: 1px solid $light_grey;
+        margin-bottom: 15px;
+    }
+    &__event-aspects{
+        display: flex;
+    }
+    &__time-img{
+        margin: 4px;
+    }
+    &__event-text{
+        min-width: 157px;
+        margin-left: 15px;
+        margin-right: 10px;
+        font-size: 16px;
+        font-weight: 500;
+        color: $purple;
+        margin-bottom: 10px;
+    }
+    &__event-text-date{
+        color: $black;
+    }
+
+    &__btn-mobile{
+        height: 48px;
+        width: 215px;
+        border: 1px solid $light_grey;
+        background-color: $light_grey2;
+        padding-left: 30px;
+        padding-right: 30px;
+        padding-top: 15px;
+        padding-bottom: 15px;
+        font-size: 14px;
+        font-weight: 700;
+    }
+    &__image-right{
+        display: none;
+    }
+    &__image-right-img{
+        width: 100%;
+        height: 100%;
+    }
+}
+
+@media (min-width: 1070px){
+    .post{
+        &__description{
+            background-color: $light_grey2;
+            padding-top: 80px;
+            padding-bottom: 65px;
+            padding-left: 84px;
+            padding-right: 84px;
+        }
+        &__description-title{
+            margin-bottom: 30px;
+        }
+        &__title{
+            font-size: 46px;
+        }
+        &__subtitle{
+            font-size: 14px;
+            margin-bottom: 30px;
+        }
+        &__description-aspects{
+            margin-bottom: 0px;
+        }
+        &__event{
+            $this: &;
+
+            &:first-child{
+                #{$this}-text-date{
+                    visibility: hidden;
+                    &::before{
+                        content: "- Thursday 23 June 2022";
```

Comment:
Not a good idea to set content in CSS, we always assume it should be manageable in CMS, so need to set it in HTML